src/ArcCommander/ArgumentProcessing.fs

```diff
@@ -33,40 +60,51 @@ module ArgumentProcessing =
         | _ -> true
    
     /// Returns true, if a value in the array contains the Mandatory attribute, but is empty
-    let private missingMandatoryAttribute (parameters : 'T []) =
-        parameters
-
-        |> Seq.exists (fun p ->
-            let unionCaseInfo,s = FSharpValue.GetUnionFields(p,typeof<'T>)
-            containsCustomAttribute<MandatoryAttribute> unionCaseInfo
-            &&
-            s = [|box ""|]
+    let private isMissingMandatoryAttribute (arguments:(string*AnnotatedArgument) []) =
```

Comment:
is -> contains

Reply:
fixed in 13ea031c0b684858a914dc93ae1d911d04ecbbe1